cpp/velox/compute/VeloxInitializer.cc

```diff
@@ -51,27 +51,6 @@ void VeloxInitializer::Init(std::unordered_map<std::string, std::string>& conf)
 
 #ifdef VELOX_ENABLE_HDFS
   velox::filesystems::registerHdfsFileSystem();
```

Comment:
we have same logical at [DwrfDatasource.cc#L110](https://github.com/oap-project/gluten/blob/main/cpp/velox/compute/DwrfDatasource.cc#L110). Does this need to change?